public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -43,6 +43,21 @@ const createActionContext = (context, action) => ({
   },
 });
 
+export const checkForError = (response, error) => {
+  for (const trigger_name in response.resp.trigger_results) {
+    // Check for errors in the trigger response
+    if (!response.resp.trigger_results[trigger_name].error) {
+      // Check for errors in the actions configured
+      for (const action_result in response.resp.trigger_results[trigger_name].action_results) {
+        error = response.resp.trigger_results[trigger_name].action_results[action_result].error;
```

Comment:
Is it possible for 1 result to have an error and not the other? This causes the second run of the for loop to hide the error

Reply:
This is from the Test Message Button Ashish! Only single action is provided as an input. The response is rendered as a list because the response from actual monitor run can contain several actions. We are sure this will contain single action result.